.gitignore

```
@@ -2,3 +2,4 @@ node_modules
 guide/.vuepress/.cache
 guide/.vuepress/.temp
 guide/.vuepress/dist
+.vscode/settings.json
```

```suggestion
.vscode/settings.json

```

Newline after files, please! :)